mk/quiet.mk

```diff
@@ -78,6 +80,7 @@ XBUILD_VERBOSITY_QUIET=/verbosity:diagnostic
 MMP_VERBOSITY=-vvvv
 MTOUCH_VERBOSITY=-vvvv
 MDTOOL_VERBOSITY=-v -v -v -v
+XIBUILD_VERBOSITY=-v
```

Comment:
should not it be something like
```
XIBUILD_VERBOSITY=$(XBUILD_VERBOSITY)
```
since that's what `MSBUILD_VERBOSITY` does and we want the same levels ?

IOW when do `xibuild` and `msbuild` verbosity need to be different ? I see that they can be, being different tools, but it seems likely it could easily confuse anyone wanting to change one.

Reply:
`XIBUILD_VERBOSITY` is to make `xibuild` itself verbose, not whatever tool it invokes.

So in theory you could construct something like this and it would make sense:

```
$(SYSTEM_XIBUILD) $(XIBUILD_VERBOSITY) -- $(MSBUILD_VERBOSITY) foo.sln
```

which means make *both* `xibuild` and `msbuild` verbose.

That said, it's implemented so that `SYSTEM_XIBUILD` automatically contains the verbosity whenever requested (which is when you pass V=1 to make), so makefiles shouldn't ever need to use `XIBUILD_VERBOSITY`.

Also `xibuild` and `msbuild` have different verbosity syntax (`-v` vs `/verbosity:...`), so they're not interchangeable.

Reply:
I get the why :) but I'm fairly sure it will cause confusion (because we'll forget) when people set the wrong one. Considering `xibuild` is a smaller wrapper I'm not sure how much value we get (now) versus the future confusion.

As for the different syntax it would be easy to have `xibuild` accept both (so the one for `msbuild` would be usable/shareable).